src/languages/en-US.ts

```diff
@@ -316,6 +316,9 @@ export default class extends Language {
 			FOOTER: this.randomDftba,
 			TITLE: 'Statistics'
 		},
+		COMMAND_DISCORD_STAUTS_DECRIPTION: 'See the current status of Discord.',
+		COMMAND_DISCORD_STAUTS_EMBED_DECRIPTION: (incident): string => `[Discord Status](https://discordstatus.com/)\n**Current Incident:**\n${incident}`,
+		COMMAND_DISCORD_STATUS_EMBED_FOOTER: (time): string => `Last changed: ${time} • ${this.randomDftba}`,
```

Comment:
Embed internationalization has a standard format; you can see how it looks and how it should be implemented with the stats command.

Reply:
Resolved by [24c4825](https://github.com/tuataria/steve/pull/51/commits/24c482586f52b18c33020abb67695e8921815edb)